data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -69,41 +63,24 @@
 	private Integer processId;
 	private Long instanceId;
 	private String defaultHiveDbName;
-
-	//	private int nextId = 0;
+	private LineageQuery lineageQuery;
 	private int subquerySeq = 0;
 	private PersistenceUnit persistenceUnit;
 
-	public static void main(String[] args) throws IOException, ParseException,
-			SemanticException, Exception {
+	public static void lineageMain(LineageQuery lineageQuery,String args1, String args2, String args3) throws ParseException, SemanticException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.apache.hadoop.hive.ql.parse.ParseException, org.apache.hadoop.hive.ql.parse.SemanticException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1160)